src/client/primitives/base.py

```diff
@@ -285,6 +292,16 @@ def unstructure(self) -> Optional[dict]:
     @classmethod
     def structure(cls, data: Any):
 
+        type_var_mapping = {}
+        if is_generic(cls):
+            base = get_origin(cls)
+            type_var_mapping = _generate_mapping(cls, type_var_mapping)
+            cls = base
+        for base in getattr(cls, "__orig_bases__", ()):
+            if is_generic(base) and not str(base).startswith("typing.Generic"):
+                type_var_mapping = _generate_mapping(base, type_var_mapping)
+                break
```

Comment:
Let's reimplement `_generate_mapping` in toloka.util, it is risky to depend on private API in core logic. Probably all this code block can be extracted to a function